apps/web/screens/Category/Category/Category.tsx

```diff
@@ -352,39 +354,31 @@ const Category: Screen<CategoryProps> = ({
                     </Text>
                   )}
                   <Stack space={2}>
-                    {sortedArticles.map(
-                      ({
-                        __typename: typename,
-                        title,
-                        slug,
-                        processEntry,
-                        processEntryButtonText,
-                      }) => {
-                        return (
-                          <FocusableBox key={slug} borderRadius="large">
-                            <TopicCard
-                              href={
-                                linkResolver(
-                                  typename?.toLowerCase() as LinkType,
-                                  [slug],
-                                ).href
-                              }
-                              // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-                              // @ts-ignore make web strict
-                              tag={
-                                (!!processEntry || processEntryButtonText) &&
-                                n(
-                                  processEntryButtonText || 'application',
-                                  'Umsókn',
-                                )
-                              }
-                            >
-                              {title}
-                            </TopicCard>
-                          </FocusableBox>
-                        )
-                      },
-                    )}
+                    {sortedArticles.map((article) => {
+                      return (
+                        <FocusableBox key={article.slug} borderRadius="large">
+                          <TopicCard
+                            href={
+                              linkResolver(
+                                article.__typename?.toLowerCase() as LinkType,
+                                [article.slug],
+                              ).href
+                            }
+                            // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                            // @ts-ignore make web strict
+                            tag={
+                              hasProcessEntries(article as Article) &&
```

Comment:
This page also doesn't display the "Skref fyrir skref" tag anymore

Ísland.is
![image](https://github.com/island-is/island.is/assets/43557895/45d3a867-d581-4452-ab51-c7168a882e42)

This PR
![image](https://github.com/island-is/island.is/assets/43557895/2874f068-022d-4b2f-ac83-2f03aa8ca204)

Reply:
In the meantime, I am checking additionally for the buttonText presence. I do think this should be address in a later PR in a more meaningful way